content/en/agent/guide/cluster-agent-custom-metrics-server.md

```diff
@@ -0,0 +1,322 @@
+
+default       nginx-6757dd8769-5xzp2                   1/1       Running   0          3m
+
+SVC:
+
+NAMESPACE     NAME                  TYPE        CLUSTER-IP        EXTERNAL-IP   PORT(S)         AGE
+default       nginx                 ClusterIP   192.168.251.36    none          8090/TCP        3m
+
+
+HPAS:
+
+NAMESPACE   NAME       REFERENCE          TARGETS       MINPODS   MAXPODS   REPLICAS   AGE
+default     nginxext   Deployment/nginx   0/9 (avg)       1         3         1        3m
+
+```
+
+### Stressing your service
+
+At this point, the setup is ready to be stressed.
+As a result of the stress, Kubernetes autoscales the NGINX pods.
+
+If you curl the IP of the NGINX service as follows:
+`curl <nginx_svc>:8090/nginx_status`, you should receive an output like:
+
+```
+$ curl 192.168.254.216:8090/nginx_status
+
+Active connections: 1 
+server accepts handled requests
+ 1 1 1 
+Reading: 0 Writing: 1 Waiting: 0 
+```
+
+Behind the scenes, the number of requests per second also increased. 
+This metric is being collected by the Node Agent, as it Autodiscovered the NGINX pod through its annotations (for more information on how Autodiscovery works, see the [Autodiscovery documentation][7]).
+Therefore, if you stress it, you should see the uptick in your Datadog app.
+As you referenced this metric in your HPA manifest, the Datadog Cluster Agent is also pulling its latest value regularly.
+Then, as Kubernetes queries the Datadog Cluster Agent to get this value, it notices that the number is going above the threshold and autoscales accordingly.
+
+To do this, run: `while true; do curl <nginx_svc>:8090/nginx_status; sleep 0.1; done`
+
+You should soon see the number of requests per second spiking and going above 9, the threshold over which the NGINX pods autoscale.
+Then, you should see new NGINX pods being created:
+
+```
+PODS:
+
+NAMESPACE     NAME                                     READY     STATUS    RESTARTS   AGE
+default       datadog-cluster-agent-7b7f6d5547-cmdtc   1/1       Running   0          9m
+default       nginx-6757dd8769-5xzp2                   1/1       Running   0          2m
+default       nginx-6757dd8769-k6h6x                   1/1       Running   0          2m
+default       nginx-6757dd8769-vzd5b                   1/1       Running   0          29m
+
+HPAS:
+
+NAMESPACE   NAME       REFERENCE          TARGETS       MINPODS   MAXPODS   REPLICAS   AGE
+default     nginxext   Deployment/nginx   30/9 (avg)     1         3         3         29m
+
+```
+
+
+### Troubleshooting
+
+- Make sure you have the Aggregation layer and the certificates set up as per the requirements section.
+- Always make sure the metrics you want to autoscale on are available.
+As you create the HPA, the Datadog Cluster Agent parses the manifest and queries Datadog to try to fetch the metric.
+- If there is a typographic issue with your metric name or if the metric does not exist within your Datadog application, the following error is raised:
```

Comment:
I don't understand the why of the bullet point here after the "normal" sentence at line # 199